tests/performance-test/deploy/README.md

```diff
@@ -2,96 +2,100 @@
 
 ## Environment
 
-minishift v1.34.1
+openshift v3.11.135
 docker v17.05+
 
 ### Setup
-SAF must already be deployed on a local minishift with the registry-route addon
-enabled. A quick way to do this is using the `quickstart.sh` script in
-`telemetry-framework/deploy/` directory to run SAF upstream version (quickstart
- can also be used to deploy the downstream):
+
+SAF must already be deployed, and your openshift must have an external route
+for the registry. A quick way to do this is using the `quickstart.sh` script in
+`telemetry-framework/deploy/` directory to run SAF. Here is an example of how
+to do that in minishift:
 
 ```shell
-$ minishift addons enable registry-route   # Run BEFORE starting minishift
-$ minishift start
-$ eval $(minishift oc-env)
-$ cd $WORKDIR/telemetry-framework/deploy/; ./quickstart.sh
+minishift addons enable registry-route   # Run BEFORE starting minishift
+minishift start
+eval $(minishift oc-env)
+cd $WORKDIR/telemetry-framework/deploy/; ./quickstart.sh
 ```
-More details about deploying SAF on Minishift can be found in the
+
+More details about deploying SAF can be found in the
 [SAF deployment docs](../../../deploy/)
 
-The Minishift registry needs to be configured such that a local docker image can
-be pushed to it. To do this, a new account must be created on Minishift that has
+The registry needs to be configured such that a local docker image can
+be pushed to it. To do this, a new account must be created on that has
```

Comment:
needs adjusted wording...

"To do this, a new account must be created on the registry that has admin privileges."